.github/workflows/verify-release-workflow.yaml

```diff
@@ -0,0 +1,24 @@
+name: Verify release workflow updates
+on: [pull_request]
+jobs:
+  check-release-changes:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Get changed files
+      # we need continue on error because the git diff | grep pipe can return a non-zero error code if no result is found
+      continue-on-error: true
+      id: changed-files
+      run: |
+            echo "changed_files=$(git diff --name-only main --  .github/workflows/ | grep releases | xargs)" >> $GITHUB_OUTPUT
```

Comment:
where do we used `changed_files`? so we mean to block subsequent actions on it's presence?

Reply:
woops, unnecessary copy/paste from the other action I based this one on, removing thins